examples/LaserWakefield/cmakeFlags

```diff
@@ -41,6 +41,7 @@ flags[8]="-DCUDA_ARCH=sm_20 -DPARAM_OVERWRITES:LIST=-DPARAM_DIMENSION=DIM2"
 flags[9]="-DCUDA_ARCH=sm_20 -DPARAM_OVERWRITES:LIST=-DPARAM_CURRENTSOLVER=currentSolver::Esirkepov<UsedParticleShape> ;-DPARAM_PARTICLESHAPE=particles::shapes::CIC;-DPARAM_DIMENSION=DIM2"
 flags[10]="-DCUDA_ARCH=sm_20 -DPARAM_OVERWRITES:LIST=-DPARAM_PRECISION=precision64Bit;-DPARAM_DIMENSION=DIM2"
 flags[11]="-DCUDA_ARCH=sm_35 -DPARAM_OVERWRITES:LIST=-DPARAM_GASPROFILE=gasNone;-DENABLE_CURRENT=0;-DPARAM_DIMENSION=DIM2"
+flags[12]="-DCUDA_ARCH=sm_20 -DPARAM_OVERWRITES:LIST=-DPARAM_GASPROFILE=gasSphereFlanks;-DENABLE_IONS=1;-DENABLE_IONIZATION=1"
```

Comment:
pls use `sm_35` and `PARAM_IONS` as described [here](https://github.com/n01r/picongpu/pull/6#discussion_r23889175)